stacks/caip10.md

```diff
@@ -0,0 +1,57 @@
+---
+namespace-identifier: stacks-caip10
+title: Stacks Namespace - Addresses
+author: Léo Pradel @pradel, Greg Ogunyanwo @gregogun
+discussions-to: https://forum.stacks.org/t/caip-2-and-caip-10-stacks-specification/13290
+status: Draft
+type: Standard
+created: 2022-06-07
+requires: "CAIP-10"
+---
+
+# CAIP-10
+
+*For context, see the [CAIP-10][] specification.*
+
+## Rationale
+
+Stacks aligns squarely with the "account" model rather than the "UXTO" model but
+uses the format "c32check" to address the "account" model.
+
+## Syntax
+
+Address string concatenates 2 fields without whitespace or dividers: a 1-byte version prefix and a 20-byte hash of any one of the following:
+- A single secp256k1 public key (either compressed or uncompressed)
+- A Bitcoin p2sh multisig script
+- A Bitcoin p2wpkh-p2sh script
+- A Bitcoin p2wsh-p2sh script
```

Comment:
Missing empty line here at the end of the list.

Reply:
good catch, thank you